Q/Qhull/build_tarballs.jl

```diff
@@ -3,14 +3,12 @@
 using BinaryBuilder
 
 name = "Qhull"
-version = v"8.0.2"
+version = v"8.1-alpha1"
```

Comment:
I actually don't think prerelease versions can be tagged.  I think we need to just call this `8.1`.

Reply:
Then what do we do when the actual 8.1 release is put out later?

Reply:
Any ideas?

Reply:
We just fundamentally don't support registering pre-release software.  You can publish this as `v8.0.999`, or `v8.1.0` and then when `v8.1.0` final is released, we delete that registration in General and have it point to the new JLL that gets deployed, or even just don't register this version and keep it only for internal testing via `pkg> add QHull_jll#v8.1.0-alpha1`.